libs/cardano-ledger-core/testlib/Test/Cardano/Ledger/Core/Arbitrary.hs

```diff
@@ -888,6 +891,39 @@ instance Arbitrary Prices where
 instance Arbitrary CostModel where
   arbitrary = elements nonNativeLanguages >>= genValidCostModel
 
+instance Era era => Arbitrary (Data era) where
+  arbitrary = Data <$> arbitrary
+
+instance Era era => Arbitrary (BinaryData era) where
+  arbitrary = dataToBinaryData <$> arbitrary
+
+instance
+  Era era =>
+  Arbitrary (Datum era)
+  where
```

Comment:
Don't know why, but that vertical style formatting with one word per line always bugs me 
```suggestion
instance Era era => Arbitrary (Datum era) where
```

Reply:
I agree, it's horrible